app/assets/stylesheets/boards.scss

```diff
@@ -0,0 +1,27 @@
+ 	.board {
+ 		width: 60%;
+    	height: 70%;
+    	border: 3px solid #333;
+    	margin: 10px auto 10px auto;
+
+ 	}
+ 	body {
+ 		 padding-top: 10%; 
+ 		 padding-bottom: 10%;
+ 		
+ 	}
+	td {
+ 		empty-cells: show;
+ 		height: 80px;
+ 		width: 80px;
+ 		vertical-align:middle;
+ 	}
+ 	table.one {
+ 		border-collapse: collapse;
+ 	}
+ 	.board tr.odd td:nth-child(odd){
+ 		  background-color: #000608
+ 	}
+ 	.board tr.even td:nth-child(even){
+ 		background-color: #000608;
+ 	}
```

Comment:
The ordering of black and white squares that results from this CSS is actually 90 degrees off from what a standard chessboard looks like. Whether you're playing as black or white, the lower left-hand corner should always be black, not white: https://en.wikipedia.org/wiki/Chessboard